### ISSUE TITLE
isatioctl doesn't honour helm api option values.kiali.prometheusAddr 

### ISSUE THREAD

**Issue by infa-ddeore** [istio/istio#22990]
**Bug description**
`kiali.prometheusAddr` from https://istio.io/docs/reference/config/installation-options/ doesn't work with `istioctl` 

1.5.1

**Expected behavior**
`values.kiali.prometheusAddr` should be honoured 

**Steps to reproduce the bug**
kiali takes `prometheus.istio-system`  instead of `prometheus.default`

```
istioctl manifest generate --set addonComponents.kiali.enabled=true --set values.kiali.prometheusAddr=http://prometheus.default:9090 | grep :9090
        url: http://prometheus.istio-system:9090
```

**Version (include the output of `istioctl version --remote` and `kubectl version` and `helm version` if you used Helm)**
```
istioctl version --remote=false
1.5.1
```

**How was Istio installed?**
downloaded the release archive from github

**Environment where bug was observed (cloud vendor, OS, etc)**
MacOs, not sure if this is same for other distros

**Comment by kharouny** [comment]
Looks like this was changed to reference the value set under `global.prometheusNamespace`

https://github.com/istio/istio/blob/master/manifests/profiles/default.yaml

Although I did like having the freedom to specify the URL that I want. For example my prometheus service is called prometheus-server.svc not prometheus.svc

**Comment by morvencao** [comment]
The fix should be available in 1.5.2.

```
# out/linux_amd64/istioctl manifest generate --set addonComponents.kiali.enabled=true --set values.kiali.prometheusAddr=http://prometheus.default:9090 | grep :9090
        url: http://prometheus.default:9090
# out/linux_amd64/istioctl version --remote=false
2020-04-18T08:05:57.240360Z	warn	Config not found: /root/.kube/config
287e52ea3c53f7a15853db550edef4dbe243b4ca
```

**Comment by infa-ddeore** [comment]
> Looks like this was changed to reference the value set under `global.prometheusNamespace`
> 
> https://github.com/istio/istio/blob/master/manifests/profiles/default.yaml
> 
> Although I did like having the freedom to specify the URL that I want. For example my prometheus service is called prometheus-server.svc not prometheus.svc

correct, `global.prometheusNamespace` needs service name to be `prometheus`, another big problem is that the bug #20059 (which will be fixed in 1.6) so we need to manually edit generated manifest and remove namespace creation object for prometheus.
